Sources/Crossroad/Router.swift

```diff
@@ -3,21 +3,13 @@ import Foundation
 public typealias SimpleRouter = Router<Void>
 
 public final class Router<UserInfo> {
-    public let scheme: String
+    private let scheme: String
```

Comment:
Should we conceal this property?

Reply:
Maybe, I guess that nobody use `Router.scheme` from external. 🤔

Reply:
I think so. I think it might be good to release API, once it becomes useful.